### DISCUSSION TITLE
Do you know why I am missing some lines in rendered image?

### DISCUSSION THREAD

**Discussion by tombohub** [mingrammer/diagrams#1028] [Q&A] [answered]
Windows 10,
graphviz 12.1.2
digrams 0.23.4

I took example from website:

```python
from diagrams import Diagram
from diagrams.aws.compute import EC2
from diagrams.aws.database import RDS
from diagrams.aws.network import ELB

with Diagram("Grouped Workers", show=False, direction="TB"):
    ELB("lb") >> [EC2("worker1"),
                  EC2("worker2"),
                  EC2("worker3"),
                  EC2("worker4"),
                  EC2("worker5")] >> RDS("events")
```

But the image is missing arrow lines:

![image](https://github.com/user-attachments/assets/5bf5b518-e242-47b3-8fbc-bf311d3f8e00)

Do you have an idea what could it be?

**Comment by mingrammer** [comment, marked as answer]
There is an issue with GraphVIz@12.x. Please downgrade the GraphViz to less than 12.x. 

Related issue comment: https://github.com/mingrammer/diagrams/issues/1019#issuecomment-2359945926
